app/controllers/AmendsController.scala

```diff
@@ -222,4 +207,52 @@ class AmendsController @Inject() (
       }
   }
 
+  private def fetchFixedProtection2016(nino: String)(implicit hc: HeaderCarrier): Future[Option[ProtectionModel]] =
+
+    fetchProtections(nino).map {
+
+      case Right(transformedReadResponseModel: TransformedReadResponseModel) =>
+        filterActiveFixedProtection2016(transformedReadResponseModel)
+
+      case Left(_) =>
+        logger.warn("Unable to retrieve Protection from GET point")
+        None
+    }
+
+  private def fetchProtections(
+      nino: String
+  )(implicit hc: HeaderCarrier): Future[Either[PlaConnectorError, TransformedReadResponseModel]] =
+    plaConnectorV2.readProtections(nino).map(_.map(TransformedReadResponseModel.from))
```

Comment:
You have to add the same conditionality as in `ReadProtectionsController` - `plaConnectorV2` should only ba called when HIP Migration feature flag is enabled. Otherwise, `plaConnector` should be called.

Reply:
New notification IDs are not guaranteed to come after HIP migration and therefore we should treat them as independent changes.